core/src/main/java/org/moditect/commands/GenerateModuleInfo.java

```diff
@@ -187,78 +187,78 @@ private void updateModuleInfo(Map<String, Boolean> optionalityPerModule, ModuleD
             moduleDeclaration.setOpen( true );
         }
 
-        List<ModuleRequiresStmt> requiresStatements = moduleDeclaration.getNodesByType( ModuleRequiresStmt.class );
+        List<ModuleRequiresDirective> requiresStatements = moduleDeclaration.findAll( ModuleRequiresDirective.class );
 
```

Comment:
"findAll" is recommended because it's part of a new family of tree-walking methods with consistent naming.